biz.aQute.bndlib/src/aQute/bnd/annotation/metatype/packageinfo

```diff
@@ -1 +1 @@
-version 1.45.0
+version 1.46.0
```

Comment:
Why bump the package minor version? The final Strings added to Configurable don't need to be public.